lms/djangoapps/course_api/api.py

```diff
@@ -268,3 +269,65 @@ def get_course_run_url(request, course_id):
     """
     course_run_url = reverse('openedx.course_experience.course_home', args=[course_id])
     return request.build_absolute_uri(course_run_url)
+
+
+def get_course_member_queryset(course_key, include_students=True, access_roles=None, prefetch_accessroles=False, prefetch_enrollments=False):
```

Comment:
I'm wondering if we can get to a simplified version of this.

Can you squash `prefetch_accessroles` and `prefetch_enrollments` down to a single parameter (`prefetch_user_role_data`)?

Reply:
Moved into one single flag ``prefetch_user_course_roles``.